src/utils.js

```diff
@@ -48,7 +48,11 @@ export function mapPathString(nodePath, state) {
     return;
   }
 
-  const modulePath = getRealPath(nodePath.node.value, state);
+  const sourcePath = nodePath.node.value;
+  const currentFile = state.file.opts.filename;
+  const opts = state.opts;
+
+  const modulePath = resolvePath.call(state, sourcePath, currentFile, opts);
```

Comment:
I'm not sure I'm a fan of using `resolvePath` with a state context.

I believe I'd prefer that the function we export for the external use would be a new one that normalize and then use the internal `resolvePath`.

In a way, it feels weird to have `resolvePath` requiring to be called on a speficic context, it should be stateless.